### DISCUSSION TITLE
How are you all using AI in and out of class?

### DISCUSSION THREAD

**Discussion by MetzinAround** [community/community#116705] [GitHub Education]
### Hi everyone,

So Ai is everywhere, but I was wondering how it's being used inside your classrooms (if at all) and how you use it for your own work (homework, projects, internships, etc.)

This can be Copilot, ChatGPT, Dalle (and other art AI), whatever. Just kind of wondering what students tend to use it for. 
Is it
- to get unstuck
- to find an annoying answer
- to write tests
- to write the code and tests
- for presentations (generative pictures and stuff)
- to straight up get the answer to some ridiculous assignment?
- to get an answer to an assignment that you were not really supposed to us AI for and you know it but you did it anyway and are hoping to not get caught (you devil, you. 🤣)

Just kind of thinking about the state of AI in education in general.

**Comment by 4ksr** [comment]
> ### Hi everyone,
> 
> So Ai is everywhere, but I was wondering how it's being used inside your classrooms (if at all) and how you use it for your own work (homework, projects, internships, etc.)
> 
> This can be Copilot, ChatGPT, Dalle (and other art AI), whatever. Just kind of wondering what students tend to use it for. Is it
> 
>     * to get unstuck
> 
>     * to find an annoying answer
> 
>     * to write tests
> 
>     * to write the code and tests
> 
>     * for presentations (generative pictures and stuff)
> 
>     * to straight up get the answer to some ridiculous assignment?
> 
>     * to get an answer to an assignment that you were not really supposed to us AI for and you know it but you did it anyway and are hoping to not get caught (you devil, you. 🤣)
> 
> 
> Just kind of thinking about the state of AI in education in general.

# Response

Hi,

It's great to see your interest in exploring how AI is being utilized in educational settings! AI indeed offers a wide range of possibilities for both students and educators.

In our classrooms, AI is increasingly becoming integrated into various aspects of learning and research. Students often leverage AI tools such as Copilot, ChatGPT, and DALL·E for a variety of purposes, including:

- Getting unstuck: AI can provide valuable insights and suggestions when students encounter challenges or need assistance with complex problems.
- Finding solutions to difficult assignments: AI tools can help students generate ideas, solve equations, or even provide code snippets for programming tasks.
- Writing code and tests: AI-powered assistants like Copilot can assist students in writing code and creating test cases for software development projects.
- Enhancing presentations: Students may use AI to create generative artwork, generate interactive visualizations, or even compose music for presentations and projects.
- Exploring creative possibilities: AI art generation tools like DALL·E allow students to explore new creative avenues and experiment with innovative forms of expression.
- Seeking answers discreetly: While not encouraged, some students may resort to using AI to find answers to assignments they're struggling with, even if it goes against academic integrity guidelines.

Overall, AI is playing an increasingly significant role in education, empowering students to explore new concepts, solve complex problems, and unleash their creativity in innovative ways. As the field of AI continues to evolve, it's exciting to see how it will continue to shape the future of education.

If you have any further questions or insights to share, please feel free to continue the discussion!

Best regards,
ksr

**Reply by samus-aran** [reply]
Oh the irony. 😉 

Hi @4ksr , 

We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. 

**Thank you for helping us maintain an authentic and beneficial space for everyone.** 👈 👈 

**Thank you for helping us maintain an authentic and beneficial space for everyone.** 👈 👈  (adding it twice so others can re-read) :)

**Reply by MetzinAround** [reply]
Absolutely stunning example of how using AI leads to content that doesn't actually answer the question! 😉 The AI model used just reframed my initial question and said "Yeah! It can be used for this!"

[Gandalf BIG WIZARD voice] 🧙 LET THIS BE A LESSON TO YOU ALL STUDENTS! 🧙

ChatGPT will let you down when used poorly.

I am SO TEMPTED to mark that as the answer though...
